nixos/modules/services/misc/heisenbridge.nix

```diff
@@ -23,7 +23,7 @@ let
 in
 {
   options.services.heisenbridge = {
-    enable = mkEnableOption "the Matrix<->IRC bridge";
+    enable = mkEnableOption "A bouncer-style Matrix IRC bridge";
```

Comment:
Don't confuse the enable option with a module description: https://github.com/NixOS/nixpkgs/blob/master/lib/options.nix#L98

Reply:
Do you have a suggestion for a better string?

Reply:
No, I like it the way it is. Almost the same formulation as the matrix-appservice-irc btw.

Reply:
No, you misunderstood ^^ With "the way it is" I meant the state of nixpkgs, not the diff :D

Reply:
Ok. The way it was before broke building the manual because `<` & `>` were interpreterd as XML.
If you have a better suggestion that doesn't break the manual I'd be happy to implement it.

Reply:
Oh, now I understand. Are XML escapes within the documentation a thing?

Reply:
I pushed a suggestion in [`4b165e7` (#154659)](https://github.com/NixOS/nixpkgs/pull/154659/commits/4b165e7675f27efe8e4ac0fabed4faa2dc3d3492)